docs/developer/validation-rules/x-ms-enum.md

```diff
@@ -0,0 +1,5 @@
+# ValidFormats
```

Comment:
This title doesn't seem to match the content? could you add the ID at the top? similar to what's in https://github.com/Azure/autorest/commit/e225e61e36644ba9119aa4996e2d3554cbe2c78c